### DISCUSSION TITLE
Support PDM - Vote for PDM support in PyCharm

### DISCUSSION THREAD

**Discussion by justasojourner** [pdm-project/pdm#617] [Ideas]
Hello all

There is a feature request for support for PDM in PyCharm raised by a JetBrains staff member.

https://youtrack.jetbrains.com/issue/PY-49588

Anyone can vote for it, you don't have to be a PyCharm user or JetBrains customer, you can connect with Google/GitHub etc.

Getting PDM recognised in PyCharm will go a long way to increasing mind share for it.

**Comment by frostming** [comment]
There are some points that are failed to be mentioned and clarified in PEP 582. One of them is the path scheme for executable files. Participate in the discussion at https://discuss.python.org/t/pep-582-python-local-packages-directory/963. More voices in the community should be heard by the Python core team to drive the PEP.

**Comment by KerberosMorphy** [comment]
Hi there, I was wondering if the change in PDM 2.0 (virtualenv by default) could help the integration of an official PDM Plugin in PyCharm
